charts/datahub-executor-worker/values.yaml

```diff
@@ -0,0 +1,104 @@
+global:
+  datahub:
+    gms:
+      secretRef: datahub-secret
```

Comment:
This is the access token secret right?

If yes, can we name this:

datahub-access-token-secret
datahub-access-token-secret-key

Reply:
done